trk/src/Screens/ClimbDetail/index.js

```diff
@@ -274,10 +279,22 @@ function ClimbDetail(props) {
                 </View>
               </View>
               <View style={styles.line}></View>
+              <View style={styles.descriptorsContainer}>
+                {climbData.descriptors && climbData.descriptors.map((descriptor, index) => (
+                  <TouchableOpacity
+                    key={index}
+                    style={styles.descriptorButton}
+                    onPress={() => onDefinition(descriptor)}
+                    activeOpacity={0.6} 
+                  >
+                    <Text style={styles.descriptorText}>{descriptor}</Text>
+                  </TouchableOpacity>
+                ))}
+              </View>
+
               <View style={styles.climbPhoto}>
                 {climbImageUrl ? <Image source={{ uri: climbImageUrl }} style={{ width: '100%', height: '100%' }} /> : <Text>Loading...</Text>}
```

Comment:
if the climb has descriptors, it maps as buttons each descriptor and makes them navigate to glossaryDefinition using the handler function 'onDefinition'